oauth2server/apps/tokens/tests/test_user_credentials.py

```diff
@@ -189,6 +189,29 @@ def test_account_is_locked_after_10_requests(self):
         locked_user = OAuthUser.objects.get(email='john4@doe.com')
         self.assertEqual(locked_user.account_locked(), True)
 
+    def test_should_login_with_different_case_email(self):
+        response = self.api_client.post(
```

Comment:
I realise the rest of the repo might not be like it, but can we start using Given, When, Then format. The refactoring I am doing is using it everywhere else